spec/namespaces/security.yaml

```diff
@@ -139,17 +339,65 @@ paths:
       responses:
         '200':
           $ref: '#/components/responses/security.update_audit_configuration@200'
+  /_plugins/_security/api/authtoken:
+    post:
+      operationId: security.authtoken.0
+      x-operation-group: security.authtoken
+      x-version-added: '1.0'
+      description: Returns the authorization token.
+      responses:
+        '200':
+          $ref: '#/components/responses/security.authtoken@200'
   /_plugins/_security/api/cache:
+    get:
+      operationId: security.cache.1
+      x-operation-group: security.cache
+      x-version-added: '1.0'
+      description: Not supported for cache API.
+      responses:
+        '501':
+          $ref: '#/components/responses/security.cache@501'
+    post:
+      operationId: security.cache.2
+      x-operation-group: security.cache
+      x-version-added: '1.0'
+      description: Not supported for cache API.
+      responses:
+        '501':
+          $ref: '#/components/responses/security.cache@501'
+    put:
+      operationId: security.cache.3
+      x-operation-group: security.cache
+      x-version-added: '1.0'
+      description: Not supported for cache API.
+      responses:
+        '501':
+          $ref: '#/components/responses/security.cache@501'
```

Comment:
Should the clients ignore `security.cache` group and not generate the `client.security.cache()` method then? If so, these operations should have `x-ignorable: true`

Reply:
not sure what x-ignorable does exactly, but currently these API even though exposed do not have appropriate handlers. I've opened an issue to capture these. https://github.com/opensearch-project/security/issues/4310

Should I mark them as x-ingorable?

Reply:
I've marked the operations as ignorable. Check [this](https://github.com/opensearch-project/opensearch-api-specification/pull/271/files#diff-7b266d1960855855ef31c24d1c2e983de7d60d31afd7927a04dfab0384a0362dR355) out.

Reply:
It's one of the [extensions](https://github.com/opensearch-project/opensearch-api-specification/blob/main/DEVELOPER_GUIDE.md#openapi-extensions) that we incorporate into the spec to add extra data to the spec that OpenAPI does not natively support. `x-ignorable` tells the client generators to skip that operation and not generate a method for it.